src/encoded/static/components/matrix_experiment.js

```diff
@@ -164,7 +164,7 @@ const convertExperimentToDataTable = (context, getRowCategories, getRowSubCatego
                 cells[columnIndex] = {
                     content: (
                         cellData.doc_count > 0 ?
-                            <a href={`${searchUrl}&${mappedSubCategoryQuery}&${columnCategoryType}=${encoding.encodedURIComponentOLD(colCategoryNames[columnIndex])}`} style={{ color: textColor }}>{cellData.doc_count}</a>
+                            <a href={`${searchUrl}&${mappedSubCategoryQuery}&${columnCategoryType}=${encoding.encodedURIComponentOLD(colCategoryNames[columnIndex])}&biosample_ontology.classification=${rowCategoryBucket.key}`} style={{ color: textColor }}>{cellData.doc_count}</a>
```

Comment:
I like to encodedURIComponent everything on the right side of a query-string equals to avoid the potential of a 301 redirect.